lib/rubocop/cop/rspec/multiple_expectations.rb

```diff
@@ -61,6 +61,26 @@ module RSpec
       #     end
       #   end
       #
+      # @example `AllowNested: false` (default)
+      #   # bad
+      #   describe UserCreator do
+      #     it 'raises an error' do
+      #       expect { my_code }.to raise_error(MyErrorType) do |error|
```

Comment:
In general this is a good example. However, raise_error supports matching the message, including with a regexp and an argument matcher, so a nested expectation is superfluous here.

What if you checked something like `error.record.persisted?` instead?

Reply:
I updated. What do you think?